sdks/python/test-suites/xlang/build.gradle

```diff
@@ -67,22 +67,13 @@ def ioXlang = new CrossLanguageTask().tap {
 
 // This list should include all expansion service targets in sdks/python/standard_expansion_services.yaml
 def servicesToGenerateFrom = [ioExpansionPath, gcpExpansionPath]
-def lowestSupportedVersion = project.properties['cross_language_validates_py_versions'].split(',')[0]
-List<String> dependencies = new File(project(":sdks:python:container:py${getVersionSuffix(lowestSupportedVersion)}").projectDir, "base_image_requirements.txt").text.split('\n')
-List<String> xlangGenerationDeps = []
-for (dep in dependencies){
-    if (dep.containsIgnoreCase("jinja2")){
-        xlangGenerationDeps.add(dep)
-        break
-    }
-}
 def xlangWrapperValidation = new CrossLanguageTask().tap {
     name = "xlangWrapperValidation"
     expansionProjectPaths = servicesToGenerateFrom
     collectMarker = "xlang_wrapper_generation"
     startJobServer = setupTask
     cleanupJobServer = cleanupTask
-    additionalDeps = xlangGenerationDeps
+    additionalDeps = ['Jinja2']
```

Comment:
i'd recommend using consistent versions with toml, either unpinned in both places or pinned in both places.

Reply:
Would it be okay to set the same bounds for both? ie. `pip install "Jinja2==>=2.7.1,<4.0.0"`?

Reply:
yeah, should be fine as long as we modify them consistently in both places (you can leave a comment in toml file)

Reply:
Done